src/Model/Anon.py

```diff
@@ -14,181 +16,114 @@
 
 
 ## ===================================HASH Function================================================
+def _gen_md5_and_sha1_hash(input):
```

Comment:
DRY.
extracted this as a function which was used numerous times.
allowed separation of access to the anonymised value of something from significant actions involving the patient data